dev/docker-compose.yml

```diff
@@ -178,6 +178,29 @@ services:
     volumes:
       - ../ci/jwt/:/usr/src/jwks/
 
+
+  prometheus:
+    image: prom/prometheus:v2.1.0
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
```

Comment:
Unless I'm missing something let's try and maintain the scope of the PR. I think changes to this file, and other's under the dev folder related to running prometheus locally, are not needed for this PR.